doozerlib/comment_on_pr.py

```diff
@@ -0,0 +1,136 @@
+import pprint
+import sys
+import requests
+import re
+import logging
+
+
+class CommentOnPr:
+    """
+    Comment on origin PR about the status of build, nightly, releases.
+    """
+
+    def __init__(self, job_id: int, logger=None):
+        self.art_dash_api_endpoint = "http://art-dash-server-aos-art-web.apps.ocp4.prod.psi.redhat.com/api/v1"
+        self.github_api_base_url_openshift = "https://api.github.com/repos/openshift"
```

Comment:
Ahh my bad. I thought I should use class variable since it was a class. Fixed in https://github.com/openshift/doozer/pull/736/commits/836e4c225a21911cc7606cbd5e6a6bb54ebad1fe !